packages/mobile/src/harmony-native/components/layout/Paper/Paper.tsx

```diff
@@ -83,25 +83,16 @@ export const Paper = forwardRef<View, PaperProps>((props, ref) => {
 
   const flexProps = { backgroundColor, borderRadius, shadow }
 
-  const universalStyles: StyleProp<ViewStyle> = { overflow: 'hidden' }
```

Comment:
do we not want this? I thought we do. Web Paper has it hidden.

If you do remove it, can you check on mobile collection page and track page to make sure this doesn't break the bottom border radius?

Reply:
so i noticed that the overflow blocked all the shadows on mobile... thoughts?

Reply:
Oh that's so weird. Is the shadow part of the content in RN?

Applying it to the inner flex makes sense. I don't think we ever want content leaking out of these.